packs/data/equipment.db/azarim.json

```diff
@@ -0,0 +1,143 @@
new, like-minded ally to take her up in arms against her foes.</p>\n<p>While she prefers to exist in the form of a rapier, she can use one of her actions to activate her shifting rune to take the shape of another weapon as needed.</p>\n<p>While <em>Azarim </em>prefers to be wielded by chaotic good characters, particularly those who worship Cayden Cailean, she's open-minded and eager to work with any wielder who seeks to oppose corrupting or toxic traditions in society, particularly those who work against the influence of the cult of Droskar. To wielders she doesn't approve of, she works to convince them to either change their ways or to hand her over to someone who she can work with, resorting to her ability to induce weakness in her carrier as needed when simply having them discard her is the only option left.</p>\n<p><em>Azarim </em>deals an additional [[/r {1d6}[chaotic]]]{1d6 chaotic damage} to worshippers of Droskar, to creatures allied with Droskar's faith, and to those who are under the influence of Droskar's faith-which includes the Kobold King Merlokrep for as long as he continues to wear his crown as well as most of the intelligent inhabitants of the deeper levels of Droskar's Crucible.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> <strong>Trigger</strong> A creature picks up <em>Azarim </em>or begins its turn carrying <em>Azarim</em>;</p>\n<p><strong>Effect</strong> The creature who carries Azarim becomes @Compendium[pf2e.conditionitems.Enfeebled]{Enfeebled 2}. This condition persists as long as the creature carries Azarim. A non-lawful creature can attempt to resist this effect with a @Check[type:fortitude|dc:23] save, but lawful creatures don't gain a save to resist.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">2</span> (command, Interact) <em>Azarim </em>casts 2nd-level @UUID[Compendium.pf2e.spells-srd.i35dpZFI7jZcRoBo]{Illusory Disguise} on her wielder. This effect ends immediately as soon as the wielder no longer carries Azarim.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> (command, Interact)</p>\n<p><strong>Frequency</strong> once per hour</p>\n<p><strong>Effect</strong> Azarim bolsters her user's ability to lie. She can either attempt her own Deception check to resolve her wielder's Lie, or she can assist with her wielder's attempt to Lie and grant a +2 circumstance bonus to the Deception check.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">3</span> (command, Interact)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Effect</strong> <em>Azarim </em>gains the effects of an anarchic weapon rune for 1 minute. When wielded by a character who's at least 11th level, <em>Azarim </em>becomes a +2 striking anarchic shifting rapier, and instead this activation can be applied to another weapon that's in physical contact with <em>Azarim</em>.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.f2U0pvTwqrLYyOlC]{Effect: Azarim}</p>"
```

Comment:
I can't make a suggestion on this without turning all the `@` symbols into mentions due to a github bug but there should be a space between the ability scores and the bonuses. We can also add a rule element to add the extra 1d6 damage predicated on `target:deity:droskar`, no need for a toggle on that, we can hide `self:deity:droskar` on the applicable NPCs in this adventure.